app/digests/models.py

```diff
@@ -1,16 +1,26 @@
 from django.db import models
 from django.contrib.postgres.fields import JSONField
 
+
 DIGEST_ID_FORMAT = r'^[A-Za-z0-9\-._]+$'
 
+PREVIEW = 'preview'
+PUBLISHED = 'published'
+
+DIGEST_STAGES = (
+    (PREVIEW, 'preview'),
+    (PUBLISHED, 'published'),
+)
+
 
 class Digest(models.Model):
     id = models.CharField(primary_key=True, max_length=255)
     content = JSONField()
     image = JSONField()
     impactStatement = models.TextField(blank=True, null=True)
-    published = models.DateTimeField()
+    published = models.DateTimeField(null=True)
     relatedContent = JSONField()
+    stage = models.CharField(max_length=25, choices=DIGEST_STAGES, default=PREVIEW)
```

Comment:
We need to update the schema (and through to Journal) to cater for these changes.

Reply:
Agreed, will help

Reply:
The schema is `@experimental` so it can change in place without a new version. However the `stage` new field is backward compatible with Journal, while `published` becoming optional requires the SDK and Journal to deal with this possibility first.

Reply:
Made a start [here](https://github.com/elifesciences/api-raml/pull/226).